src/effects/effectchain.cpp

```diff
@@ -95,9 +95,6 @@ EffectChainPointer EffectChain::clone(EffectChainPointer pChain) {
     pClone->setEnabled(pChain->enabled());
     pClone->setName(pChain->name());
     pClone->setMix(pChain->mix());
-    foreach (const ChannelHandleAndGroup& handle_group, pChain->enabledChannels()) {
-        pClone->enableForChannel(handle_group);
-    }
```

Comment:
Hm, looks like EffectChain::enable/disableForChannel don't check m_bAddedToEngine before sending a request to the engine and EffectChain::addToEngine doesn't also send the group enable states -- so that would explain it.

Maybe clone shouldn't set mix or enabled either since those are properties of the slot too?

Please add a comment mentioning that we don't clone the group state since it's a property of the slot.